examples/core/Graphormer/train.py

```diff
@@ -0,0 +1,493 @@
+import argparse
```

Comment:
Can you add a short summary of how Graphormer works and how the example is organized?

Reply:
example: https://github.com/dmlc/dgl/blob/master/examples/sampling/node_classification.py